samples/bluetooth/direct_test_mode/src/fem/nrf21540.c

```diff
@@ -312,34 +302,6 @@ static int gpio_configure(void)
 	return gpio_pin_configure_dt(&nrf21540_cfg.ant_sel, GPIO_OUTPUT_INACTIVE);
 }
 
-static int spi_config(void)
-{
-	int err;
-
-	/* Configure CS pin */
-	nrf21540_cfg.spi_cs.gpio_dev = device_get_binding(CS_GPIO_PORT);
-	if (!nrf21540_cfg.spi_cs.gpio_dev) {
-		return -ENXIO;
-	}
-
-	err = gpio_pin_configure(nrf21540_cfg.spi_cs.gpio_dev,
-				 nrf21540_cfg.spi_cs.gpio_pin,
-				(GPIO_OUTPUT_INACTIVE |
-				 nrf21540_cfg.spi_cs.gpio_dt_flags));
-	if (err) {
-		return err;
-	}
-
-#if !defined(NRF5340_XXAA_NETWORK)
-	nrf21540_cfg.spi_dev = device_get_binding(NRF21540_SPI_LABEL);
-	if (!nrf21540_cfg.spi_dev) {
-		return -ENXIO;
-	}
-#endif
-
-	return 0;
```

Comment:
@KAGA164 can you please check these changes? I don't fully understand why CS is used without an actual SPI device. Can you please clarify this code?